openmc/mgxs/mgxs.py

```diff
@@ -212,11 +229,70 @@ def num_groups(self):
         return self.energy_groups.num_groups
 
     @property
+    def scores(self):
+        return ['flux', self.rxn_type]
+
+    @property
+    def filters(self):
+        group_edges = self.energy_groups.group_edges
+        energy_filter = openmc.Filter('energy', group_edges)
```

Comment:
Except that filters don't have IDs. Point taken, but I'm not sure it
matters.
On May 13, 2016 12:38 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/mgxs/mgxs.py
> <https://github.com/mit-crpg/openmc/pull/644#discussion_r63213423>:
>
> > @@ -212,11 +229,70 @@ def num_groups(self):
> >          return self.energy_groups.num_groups
> >
> >      @property
> > +    def scores(self):
> > +        return ['flux', self.rxn_type]
> > +
> > +    @property
> > +    def filters(self):
> > +        group_edges = self.energy_groups.group_edges
> > +        energy_filter = openmc.Filter('energy', group_edges)
>
> Not sure if it would ever really be an issue, but looking at this property
> multiple times will result in Filters that have different ids
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/644/files/47ef320ad517612376e181ec6a6bc42ca0db98ce#r63213423>
>